src/flagStore.ts

```diff
@@ -29,45 +37,72 @@ export class FlagStore {
 		this.start();
 	}
 
-	async reload(e?: ConfigurationChangeEvent) {
+	async reload(e?: ConfigurationChangeEvent | undefined) {
```

Comment:
```suggestion
	async reload(e?: ConfigurationChangeEvent) {
```
I might have added this in our live share session and forgot to remove it